src/WebdriverClassicDriver.php

```diff
@@ -11,11 +11,1194 @@
 namespace Mink\WebdriverClassDriver;
 
 use Behat\Mink\Driver\CoreDriver;
+use Behat\Mink\Exception\DriverException;
+use Facebook\WebDriver\Exception\NoSuchCookieException;
+use Facebook\WebDriver\Exception\NoSuchElementException;
+use Facebook\WebDriver\Exception\UnsupportedOperationException;
+use Facebook\WebDriver\Exception\WebDriverException;
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\Remote\RemoteWebElement;
+use Facebook\WebDriver\Remote\WebDriverBrowserType;
+use Facebook\WebDriver\Remote\WebDriverCapabilityType;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverDimension;
+use Facebook\WebDriver\WebDriverElement;
+use Facebook\WebDriver\WebDriverPlatform;
+use Facebook\WebDriver\WebDriverRadios;
+use Facebook\WebDriver\WebDriverSelect;
+use JetBrains\PhpStorm\Language;
+use JsonException;
+use Throwable;
 
 class WebdriverClassicDriver extends CoreDriver
 {
+    public const DEFAULT_BROWSER = WebDriverBrowserType::CHROME;
+
+    public const DEFAULT_CAPABILITIES = [
+        'default' => [
+            'platform' => 'ANY',
+            'name' => 'Behat Test',
+            'deviceOrientation' => 'landscape',
+            'deviceType' => 'desktop',
+        ],
+
+        WebDriverBrowserType::CHROME => [
+            'goog:chromeOptions' => [
+                // disable "Chrome is being controlled.." notification bar
+                'excludeSwitches' => ['enable-automation'],
+            ],
+        ],
+
+        WebDriverBrowserType::MICROSOFT_EDGE => [
+            'ms:edgeOptions' => [
+                // disable "Microsoft Edge is being controlled.." notification bar
+                'excludeSwitches' => ['enable-automation'],
+                // disable menu shown when text is selected (which interferes with double-clicking)
+                'prefs' => [
+                    'edge_quick_search.show_mini_menu' => false,
+                ],
+            ],
+        ],
+    ];
+
+    private const BROWSER_NAME_ALIAS_MAP = [
+        'edge' => WebDriverBrowserType::MICROSOFT_EDGE,
+        'chrome' => WebDriverBrowserType::CHROME,
+        'firefox' => WebDriverBrowserType::FIREFOX,
+    ];
+
+    private const W3C_WINDOW_HANDLE_PREFIX = 'w3cwh:';
+
+    private ?RemoteWebDriver $webDriver = null;
+
+    private string $browserName;
+
+    private DesiredCapabilities $desiredCapabilities;
+
+    private array $timeouts = [];
+
+    private string $webDriverHost;
+
+    private ?string $initialWindowName = null;
+
+    /**
+     * @param string $browserName One of 'edge', 'firefox', 'chrome' or any one of {@see WebDriverBrowserType} constants.
+     */
+    public function __construct(
+        string $browserName = self::DEFAULT_BROWSER,
```

Comment:
@stof recently I found that the proper browser name for MSEdge is "MicrosoftEdge".
For some reason, "edge" used to work before, but not anymore.
At the moment, we use the browser name from the docker image suffix, which eventually triggered the problem I just mentioned.

However, this got me thinking, at the moment there aren't any restrictions or suggestions for the browser name from the mink side, which is far from ideal especially since we have extra (default) initialization logic based on the browser name.

On the other hand, I'm not sure it's a good idea to tell the dev to use constants from a 3rd party library (what I sort of the in the phpdoc). I took the liberty to say exactly which browsers are definitely supported in the PHPDoc, which is effectively the keys from `BROWSER_NAME_ALIAS_MAP`. Another approach could be to define our own constants, or provide them in a new class that extends `WebDriverBrowserType`.

What do you think?